### PR TITLE
Fix Thumb Drive Creation failure (>= 64GB))

### PR DESCRIPTION
**Pull request by zayedrasha**

### CODE REVIEW

File: CreateSurfaceWindowsImage.ps1 [hunk 1]
```diff
@@ -3354,7 +3354,7 @@ Function Update-Win10WIM
                     }
                     Else
                     {
-                        $NewUSBDriveLetter = New-Partition -DiskNumber $TempUSB -UseMaximumSize -AssignDriveLetter | Format-Volume -FileSystem FAT32 -NewFileSystemLabel $Device
+                        $NewUSBDriveLetter = New-Partition -DiskNumber $TempUSB -Size 14GB -AssignDriveLetter | Format-Volume -FileSystem FAT32 -NewFileSystemLabel $Device
```
Comment (luy-ce) [inline CreateSurfaceWindowsImage.ps1:3357]: Where did you get 14GB from? Why does it fix the issue?

Comment (zayedrasha) [reply CreateSurfaceWindowsImage.ps1:3357]: using -UseMaximumSize is not supported and throws error sizeNotSupported. The size should be between 14GB to 30GB to enter this block, so took the 14GB as it's the minimum size required and created the image successfully!  please let me know if you have a better solution..TIA

Comment (luy-ce) [reply CreateSurfaceWindowsImage.ps1:3357]: Gotcha. Do you know how big the contents were after successful image creation? I just want to make sure it's not too close to 14GB, as image size will always increase with time and fluctuate a little.

Comment (zayedrasha) [reply CreateSurfaceWindowsImage.ps1:3357]: Good point, let me double check that!

**Review by luy-ce — APPROVED** [review]

**Pull request merged**